rehashing how we operate

ChristinaLK opened the pull request:
Re-organizing and fleshing out some of How We Operate based on trainer discussions.  
Address #821 #909 #929 

Still needs venn diagram (maybe), info about charging, general cleaning up. 

Specific requests: 
- how much of this, if any, should be in the handbook instead, and referenced here? I'm thinking especially of the central vs self-organized workshop pieces. @maneesha 
- is the stuff about regional coordinators correct? should we list their names? @sheraaronhurt 

@karenword @raynamharris

ChristinaLK commented:
Ahh shoot, should also address #615 here. may do this now or add it after these major changes.

sheraaronhurt commented:
@ChristinaLK the verbiage that @raynamharris  created in #909 is great. If you just be sure to link to the page of RC's that would suffice. Thanks!

ChristinaLK commented on _episodes/21-carpentries.md:
```diff
@@ -150,42 +97,132 @@ and their respective logos
+| Component | Centrally-Organized | Self-Organized
+|---------- |-------------------- |-------------- |
+| Location  | The local organizer provides the space and organizes local logistics. | The local organizer provides the space and organizes local logistics. |
+| Instructors | The Carpentries organization finds instructors for the workshop; the local workshop organizer books the instructor's travel and finds local helpers to assist. | The workshop organizer finds both instructors and helpers. |
```
can express travel limitations to the Carpentries.

ChristinaLK commented on _episodes/21-carpentries.md:
```diff
@@ -18,49 +19,50 @@ keypoints:
-who share a mission to teach foundational computational and data science skills. 
+The Carpentries project comprises communities of instructors, trainers,
+maintainers, helpers, and supporters from
+[Software Carpentry]({{ site.swc_site }}), [Data Carpentry]({{ site.dc_site }}) and [Library Carpentry]({{ site.lc_site }})
```
I can change it back, I understand the reasoning. 

It was weird to me that instructors/trainers/maintainers were capitalized and helpers wasn't, but I'm guessing that helpers aren't considered official community roles?

ChristinaLK commented on _episodes/21-carpentries.md:
```diff
@@ -18,49 +19,50 @@ keypoints:
 *   a focus on technical skills.
-*   two-day format taught by volunteer instructors.
+*   an intensive one-day or two-day format taught by volunteer instructors.
 *   a focus on filling gaps in current training for learners.
```
👍

ChristinaLK commented on _episodes/21-carpentries.md:
```diff
@@ -150,42 +97,132 @@ and their respective logos
-*   it covers the core topics,
-*   at least one instructor is certified,
+*   it covers the core topics (see below),
+*   at least one of the instructors is a certified Carpentries instructor,
```
👍

ChristinaLK commented on _episodes/21-carpentries.md:
```diff
@@ -150,42 +97,132 @@ and their respective logos
+instructors and helpers at a branded workshop do not have to be certified. We
+rely on the judgment of local organizers to recruit and orient volunteers who
+will embody the Carpentries ethos and practices.
+
```
Good idea.

ChristinaLK commented on _episodes/21-carpentries.md:
```diff
@@ -150,42 +97,132 @@ and their respective logos
+* Europe
+* Africa
+* Australia/New Zealand
+
```
I may use @raynamharris suggestion which includes a link to that page.

ChristinaLK commented:
Addressed most of Karen's suggestions. Moved some of the workshop stuff to the handbook, 
https://github.com/carpentries/handbook/pull/456

Still need to add etherpad, I think. Will probably do that as a separate PR.

sheraaronhurt commented:
These are great changes. I'd only ask that we make Regional Coordinators a proper noun and capitalize their titles :) Thanks!

ChristinaLK commented:
@sheraaronhurt should be done.

@karenword @raynamharris is this ready to merge? @brownsarahm is willing to add information about the etherpad to this section, but she should probably work from this copy.

brownsarahm commented:
should I still wait to address #614 and #615?

ChristinaLK commented:
@brownsarahm maybe just go ahead and I'll work it in after the fact!

raynamharris commented:
conflicts resolved!
